src/components/confirmation-box/confirmation-box.tsx

```diff
@@ -18,6 +18,16 @@ const ConfirmationBox: React.FC<ConfirmationBoxProps> = ({
   preventAnimation,
   children,
 }) => {
+  useEffect(() => {
+    const echapPressed = (event: KeyboardEvent): void => {
+      if (event.key === "Escape") {
+        setOpen(false);
+      }
+    };
+    window.addEventListener("keyup", echapPressed);
+    return () => window.removeEventListener("keyup", echapPressed);
+  });
```

Comment:
I added a keyup event on the `Escape` keyboard key to close the confirmation box.